forge-gui/res/cardsfolder/b/barbarian_class.txt

```diff
@@ -1,7 +1,9 @@
 Name:Barbarian Class
 ManaCost:R
 Types:Enchantment Class
-S:Mode$ Continuous | Affected$ You | AddKeyword$ If you would roll one or more dice, instead roll that many dice plus one and ignore the lowest roll. | Description$ If you would roll one or more dice, instead roll that many dice plus one and ignore the lowest roll.
+R:Event$ RollDice | ValidPlayer$ You | ReplaceWith$ PlusRoll | Description$ If you would roll one or more dice, instead roll that many dice plus one and ignore the lowest roll.
```

Comment:
missing ActiveZones$

Reply:
I forgot replacement effects don't have the same zone defaults as statics